roles/routeros_common/tasks/manage_users.yml

```diff
@@ -0,0 +1,102 @@
+---
+# Manage Mikrotik Users
+# Missing users are added with random password and ssh public key loaded.
+# Users present and in the delete_users list are delete.
+# For now, we leave extra users as is.
+
+    - name: setup expected users
```

Comment:
maybe I'm demonstrating my lack of ansible or yaml expertise here, but is there a reason that this whole file starts indented 3 spaces?

Reply:
No good reason.  Largely he result of some cut/paste and not enough cleanup.  Fixed in next commit.